src/documentation/markdown/2017/lar-data-fields.md

```diff
@@ -0,0 +1,378 @@
+  - 4 - Not applicable
+  - 5 - No co-applicant
+  - 6 - Co-applicant selected both male and female  
+
+### [income](#income)
+
+- **Description:** The gross annual income, in thousands of dollars, relied on in making the credit decision, or if a credit decision was not made, the gross annual income relied on in processing the application
+- **Values:**
+  - Varying values
+
+### [purchaser\_type](#purchaser_type)
+
+- **Description:** Type of entity purchasing a covered loan from the institution
+- **Values:**
+  - 0 - Loan was not originated or was not sold in calendar year covered by register
+  - 1 - FNMA (Federal National Mortgage Association)
+  - 2 - GNMA (Government National Mortgage Association)
+  - 3 - FHLMC (Federal Home Loan Mortgage Corporation)
+  - 4 – FAMC (Federal Agricultural Mortgage Corporation)
+  - 5 - Commercial bank
+  - 6 - Savings bank or savings association
+  - 7 - Life insurance company
+  - 8 - Affiliate institution
+  - 9 - Other type of purchaser 
+
+### [denial\_reason\_1](#denial_reason_1)
+
+- **Description:** The principal reason, or reasons, for denial
+- **Values:**
+  - 1 - Debt-to-income ratio
+  - 2 - Employment history
+  - 3 - Credit history
+  - 4 - Collateral
+  - 5 - Insufficient cash (downpayment, closing costs)
+  - 6 - Unverifiable information
+  - 7 - Credit application incomplete
+  - 8 - Mortgage insurance denied
+  - 9 - Other
+
+### [denial\_reason\_2](#denial_reason_2)
+
+- **Description:** The principal reason, or reasons, for denial
+- **Values:**
+  - 1 - Debt-to-income ratio
+  - 2 - Employment history
+  - 3 - Credit history
+  - 4 - Collateral
+  - 5 - Insufficient cash (downpayment, closing costs)
+  - 6 - Unverifiable information
+  - 7 - Credit application incomplete
+  - 8 - Mortgage insurance denied
+  - 9 - Other
+
+### [denial\_reason\_3](#denial_reason_3)
+
+- **Description:** The principal reason, or reasons, for denial
+- **Values:**
+  - 1 - Debt-to-income ratio
+  - 2 - Employment history
+  - 3 - Credit history
+  - 4 - Collateral
+  - 5 - Insufficient cash (downpayment, closing costs)
+  - 6 - Unverifiable information
+  - 7 - Credit application incomplete
+  - 8 - Mortgage insurance denied
+  - 9 - Other
+
+### [rate\_spread](#rate_spread)
+
+- **Description:** The difference between the covered loan’s annual percentage rate (APR) and the average prime offer rate (APOR) for a comparable transaction as of the date the interest rate is set
+- **Values:**
+  - Varying values
+
+### [hoepa\_status](#hoepa_status)
+
+- **Description:** Whether the covered loan is a high-cost mortgage
+- **Values:**
+  - 1 - High-cost mortgage
+  - 2 - Not a high-cost mortgage
+  - 3 - Not applicable
+
+### [lien\_status](#lien_status)
+
+- **Description:** Lien status of the property securing the covered loan, or in the case of an application, proposed to secure the covered loan 
+- **Values:**
+  - 1 - Secured by a first lien
```

Comment:
for 2017:
1.Secured by a first lien
2.Secured by a subordinate lien
3.Not secured by a lien
4.Not applicable (purchased loans)